README.md

```diff
@@ -1 +1,218 @@
-# personal_badget
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
```

Comment:
- [ ] [OPTIONAL] This logo link is broken. Please make sure to fix it in order to make your readme professional and descriptive :+1: